### PR TITLE
[ARM-292] Force refresh the invitation creators telemetry after friend accepts it 

### PR DESCRIPTION
**Pull request by JasterV**
I've created a "sync_users_location" function to solve this, and also I've refactored the "force_refresh_telemetry_internal" function to be more generic

### CODE REVIEW

File: rust/src/server/invitations.rs [hunk 1]
```diff
@@ -80,6 +81,10 @@ fn accept(id: String, auth_info: AuthInfo, state: State<Storage>) -> APIResult<M
             let _ = notify_accepted(&mut conn, &data)
                 .map_err(|w| w.log_err("Error sending notification"));
 
+            let inv_creator_data = get_invitation_creator(&mut conn, &id)?;
+            let creator_username: String = inv_creator_data["username"].as_str().unwrap().into();
+            sync_users_location(&mut conn, auth_info.username.clone(), creator_username)?;
```
Comment (darioalessandro) [inline rust/src/server/invitations.rs]: @griffobeid ^^

File: rust/src/controllers/telemetry.rs [hunk 2]
```diff
@@ -419,21 +419,6 @@ pub fn username_has_follower(
     Ok(!rows.is_empty())
 }
 
-pub fn sync_users_location(
```
Comment (JasterV) [inline rust/src/controllers/telemetry.rs]: Why did you remove this function instead of refactoring it? @darioalessandro

Comment (darioalessandro) [reply rust/src/controllers/telemetry.rs]: Hi @JasterV:

so it turns out that all we needed to do is call force refresh for the invitation creator

File: rust/Cargo.lock [hunk 3]
```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
```
Comment (darioalessandro) [inline rust/Cargo.lock:3]: I did not touch this file, let me know if this is expected

File: rust/src/bin/emergency.rs [hunk 4]
```diff
@@ -1,26 +1,28 @@
-use log::debug;
 use lib::server::emergency::rocket;
 use lib::server::middleware::logging;
+use log::debug;
```
Comment (darioalessandro) [inline rust/src/bin/emergency.rs:3]: cargo fmt stuff

File: rust/src/bin/http_gateway.rs [hunk 5]
```diff
@@ -1,26 +1,28 @@
...
 
 fn main() {
     env_logger::init();
     match std::env::var("SENTRY_DSN") {
         Ok(dsn) => {
-            let sentry_logger = logger::init(dsn, Some(InitConfig {
-                service: Some("Http gateway"),
-                ..Default::default()
-            }));
+            let sentry_logger = logger::init(
```
Comment (darioalessandro) [inline rust/src/bin/http_gateway.rs:10]: cargo fmt stuff

File: rust/src/bin/invitations.rs [hunk 6]
```diff
@@ -1,26 +1,28 @@
-use log::debug;
 use lib::server::invitations::rocket;
 use lib::server::middleware::logging;
+use log::debug;
```
Comment (darioalessandro) [inline rust/src/bin/invitations.rs:3]: cargo fmt stuff

File: rust/Cargo.lock (hunk 3 above)
Comment (griffobeid) [reply rust/Cargo.lock:3]: I would be curious to see if the file is generated differently on macOS vs our docker container (linux).
